docs: Rename 'QuickStart' to 'Getting Started'

FrederikSchlemmer opened the pull request:
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x". -->
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->

Issue Number: #25524


## What is the new behavior?
In the entire project 'QuickStart' has been changed to 'Getting Started'.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information
I am not sure if it is wanted that the deletion of the symlinks is also considered as a change by Github.

I mean the deletion of 'packages/upgrade/static/src'.

trotyl commented on aio/content/guide/attribute-directives.md:
```diff
@@ -152,8 +152,8 @@ Of course you could reach into the DOM with standard JavaScript and attach event
-1. The code must *detach* the listener when the directive is destroyed to avoid memory leaks.
-1. Talking to DOM API directly isn't a best practice.
+2. The code must *detach* the listener when the directive is destroyed to avoid memory leaks.
+3. Talking to DOM API directly isn't a best practice.
```
Why this change?

sarunint replied:
@FrederikSchlemmer if you want to make this change, submit to it as another PR. This PR focuses on changing QuickStart to Getting Started.

trotyl replied:
Also this will make the maintenance harder, if you want to insert something beside `1` and `2`, then you need to change from line 2 to the end. Leaving everything to 1 make it much easier, just a little ugly in source code.

sarunint replied:
Wow, I don't know that before!

FrederikSchlemmer replied:
@trotyl @sarunint The change was not intended and actually not made by me. Maybe this place was modified by Visual Studio code?

StephenFluin reviewed (changes requested):
This is awesome! Thanks for this contribution. Can we also rename the URL  and filepath of the quickstart to be getting started as well so that we have consistency?

Does the deletion of /upgrade/static/src belong in this PR? I feel like it should be removed

FrederikSchlemmer committed:
docs: Rename 'QuickStart' into 'Getting Started'

Delete symlink

docs: Undo unwanted changes

docs: Rename 'QuickStart' into 'Getting Started'

Revert symlink commit

FrederikSchlemmer commented:
@StephenFluin As already described in 'Other Information' I wasn't sure if this deletion belongs in the commit either. I have now taken it out.

@jenniferfell My recommendation would be to just change the in-page title and leave the nav title, URL, etc. in place. There are various links to this page, which would break.
Maybe the linking problem should be treated as a separate issue?

jenniferfell commented:
Thanks for the ping. I haven't forgotten this. In fact, I've been rewriting the Getting Started for other reasons, so I changed the page title. https://github.com/angular/angular/pull/26093

I'm hoping that rewrite is merged in the next day or so. Then I'd like to merge your changes to the cross reference text. (Thanks for finding all of those.) Would that work for you? It means one more rebase, probably late this week or next. I can let you know. 

Re the URL...we're still considering in the context of some future doc work. 

Thanks again.

gkalpak commented:
I closed and reopened the PR, because Travis was stuck. Now it need re-approval 😇
